docs/make.jl

```diff
@@ -15,11 +17,12 @@ makedocs(
     sitename = "MIRT.jl",
     pages    = [
         "Home" => "index.md",
+        "Table of Contents" => "toc.md",
         # "Examples" => examples,
         "Function References" => "reference.md",
     ]
 )
 
 # examples_cb()
 
-deploydocs(repo   = "github.com/JeffFessler/MIRT.jl")
+deploydocs(repo = "github.com/JeffFessler/MIRT.jl")
```

Comment:
> I have run julia make.jl locally to try to generate html documentation and it runs locally but you put /docs/build into .gitignore so it doesn't end up on github. 

This is a no-op when we run it locally, and will push to `gh-pages` branch if it's running on a CI environment, e.g., Github Actions. See also: https://juliadocs.github.io/Documenter.jl/dev/man/hosting/#.gitignore

I'm really sorry that I haven't found some time to add the demos here, I was way too exhausted these days in a day-to-day seminar. My limited time for Julia is spent on setting up storage mirror servers, e.g., [SJTUG](https://mirrors.sjtug.sjtu.edu.cn/#/) and [BFSU](https://mirrors.bfsu.edu.cn/julia/) so that users in China can add packages without much trouble; they are broken these days and I spent a lot of time on debugging and enhancing the tool.